### ISSUE TITLE
Add GitHub actions to update docs repo whenever either private mirror changes

### ISSUE THREAD

**Issue by catdevnull** [buttondown/roadmap#3304]
https://github.com/buttondown/docs

see https://github.com/buttondown/roadmap/issues/2737

**Comment by catdevnull** [comment]
just to confirm, this would just be commiting the entire docs-v2 directory, right? should it exclude something?

**Comment by jmduke** [comment]
correct! i think the only file we'd want to exclude is `vectors.json`, because it's huge and noisy
